_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -0,0 +1,130 @@
+    those generating the data---but probably not by enough to
+    significantly discourage the behavior---and the alternative storage
+    methods might create new problems for traditional Bitcoin users.
+
+- **Summary of call about mitigating LN jamming:** Carla Kirk-Cohen and
+  Clara Shikhelman [posted][ckccs jamming] to the Lightning-Dev mailing
+  list a summary of a recent video conversation about attempts to
+  address [channel jamming attacks][topic channel jamming attacks].
+  Topics discussed included upgrade mechanism tradeoffs, a simple
+  proposal for upfront fees derived from a recent paper (see [Newsletter
+  #226][news226 jam]), the CircuitBreaker software (see [Newsletter
+  #230][news230 jam]), an update on reputation credentials (see
+  [Newsletter #228][news228 jam]), and related work from the Lightning
+  Service Provider (LSP) specification working group.  See the mailing
+  list post for extended summaries and a [transcript][jam xs].
+
+    Future video meetings are planned for every two weeks; watch the
+    Lightning-Dev mailing list for announcements of upcoming meetings.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME is a PR by FIXME that FIXME:LarryRuane
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/FIXME#l-FIXME"
+%}
+
+<!-- FIXME:harding to add releases/rc if any on Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25880][] p2p: Make stalling timeout adaptive during IBD FIXME:Xekyo
+
+- [Core Lightning #5679][] sql plugin FIXME:adamjonas
+
+- [Core Lightning #5821][] adds `preapproveinvoice` (pre-approve
+  invoice) and `preapprovekeysend` (pre-approve keysend) RPCs that allow
+  the caller to send either a [BOLT11][] invoice or a [keysend][topic
+  spontaneous payments] payment details to Core Lightning's signing module
+  (`hsmd`) to verify the module is willing to sign the payment.  For
+  some applications, such as those where the amount of money which can
+  be spent is rate limited, asking for pre-approval might produce fewer
+  problems than simply attempting the payment and dealing with failure.
+
+- [Core Lightning #5849][] makes backend changes that allow a node to
+  handle over 100,000 peers each with one channel.  Although someone
```

Comment:
I wasn't sure I was grokking correctly, looked at the PR, and "quickly open 100,000 peers with one channel each" in the merged pull description seems clearer to me.